### PR TITLE
Add benchmark for channel_shuffle operator

### PR DESCRIPTION
**Pull request by taivu1998**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43602 Implement ChannelShuffle op with XNNPACK
* **#43509 Add benchmark for channel_shuffle operator**

Differential Revision: [D23299972](https://our.internmc.facebook.com/intern/diff/D23299972)

### CODE REVIEW

**Review by kimishpatel — COMMENTED** [review]
Can you run the test and report the results here?

**Comment by taivu1998** [comment]
| Hyperparameters  | Time (us) |
| ------------- | ------------- |
| Batch_size: 1, Channels per group: 16, Height: 16, Width: 16, Groups: 1  | 3.990  |
| Batch_size: 2, Channels per group: 16, Height: 16, Width: 16, Groups: 2  | 10.510 |
| Batch_size: 2, Channels per group: 32, Height: 32, Width: 32, Groups: 2  | 47.635 |
| Batch_size: 4, Channels per group: 32, Height: 32, Width: 32, Groups: 4  | 735.698 |
| Batch_size: 4, Channels per group: 64, Height: 64, Width: 64, Groups: 4  | 9783.544 |

**Review by kimishpatel — APPROVED** [review]
Looks good to me.

File: benchmarks/operator_benchmark/pt/channel_shuffle_test.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import operator_benchmark as op_bench
+import torch
+
+
+"""Microbenchmarks for channel_shuffle operator."""
+
+
+# Configs for PT channel_shuffle operator
+channel_shuffle_long_configs = op_bench.cross_product_configs(
```
Comment (mingzhe09088) [inline benchmarks/operator_benchmark/pt/channel_shuffle_test.py:14]: Could you try to reduce the number of configs here? This config will create 5 x 4 x 4 x 4 x 4 x 2 = 2560 different test cases. Ideally we would like each operator to have less than 100 different tests.

**Comment by mingzhe09088** [comment]
Could you add this benchmark to https://github.com/pytorch/pytorch/blob/master/benchmarks/operator_benchmark/benchmark_all_other_test.py so it will be tested automatically?

File: benchmarks/operator_benchmark/pt/channel_shuffle_test.py (hunk 1 above)
Comment (kimishpatel) [reply benchmarks/operator_benchmark/pt/channel_shuffle_test.py:14]: Thanks @mingzhe09088. Tai, can you reduce each param to only two values?

**Pull request closed without merging**